2-copy-of-code/lesson-13/scripts/amazon.js

```diff
@@ -98,8 +111,19 @@ document.querySelectorAll('.js-add-to-cart')
 
       addedMessage.classList.add('added-to-cart-visible');
 
-      setTimeout(() => {
```

Comment:
I can comprehend why ${productId} is added. is it because of the class="" naming syntax?

Reply:
No it is used to deal with each product's timeout managing them separately